src/components/ReviewItem/RenderImages.tsx

```diff
@@ -0,0 +1,31 @@
+import type { ReviewImage } from '@/types/ProductReviewTypes';
+import classNames from 'classnames/bind';
+import Image from 'next/image';
+import styles from './ReviewItem.module.scss';
+
+const cn = classNames.bind(styles);
+
+interface RenderImagesProps {
+  reviewImgs: ReviewImage[];
+  className: string;
+  width: number;
+  height: number;
+  altPrefix: string;
+}
+
+export default function RenderImages({ reviewImgs, className, width, height, altPrefix }: RenderImagesProps) {
+  return (
+    <>
+      {reviewImgs.map((item, idx) => (
+        <Image
+          key={item.id}
+          className={cn(className)}
+          src={item.imageUrl}
```

Comment:
```suggestion
          key={item.id}
          className={cn(className)}
          src={item.imageUrl}
```
className을 내리실때 
```tsx
      <RenderImages
        className='all-review-image'
        reviewImgs={reviewImgs}
        width={228}
        height={228}
        altPrefix='전체 리뷰 이미지'
      />
      ```
     이런식으로 사용하고계신데 좋은 사용방법은 아닌거같아요